src/app/components/DesktopHeaderNav/DesktopHeaderNav.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from "react";
+import Link from "next/link";
+import { DesktopHeaderNavProps } from "./DesktopHeaderNav.types";
+import { NavIconButton } from "../Header/Header.types";
+import IconButton from "../IconButton/IconButton";
+import MainNav from "./MainNav";
+
+function DesktopHeaderNav({
```

Comment:
the most commonly used approach to defining the type or interface  for the functional react components is next
`
import { FC } from "react";

function DesktopHeaderNav: FC<DesktopHeaderNavProps>({ playerName = "Player", betOptions }) => {}`